src/cmdlib.sh

```diff
@@ -519,14 +527,15 @@ EOF
                     -kernel "${vmbuilddir}/kernel" -initrd "${vmbuilddir}/initrd" \
                     -no-reboot -nodefaults \
                     -device virtio-serial \
-                    -virtfs 'local,id=workdir,path='"${workdir}"',security_model=none,mount_tag=workdir' \
+                    -virtfs 'local,id=etcpki,path='/etc/pki',security_model=none,mount_tag=etcpki,multidevs=remap' \
+                    -virtfs 'local,id=workdir,path='"${workdir}"',security_model=none,mount_tag=workdir,multidevs=remap' \
                     -append "root=/dev/vda console=${DEFAULT_TERMINAL} selinux=1 enforcing=0 autorelabel=1" \
                    )
 
     # support local dev cases where src/config is a symlink
     if [ -L "${workdir}/src/config" ]; then
         # qemu follows symlinks
-        base_qemu_args+=("-virtfs" 'local,id=source,path='"${workdir}"'/src/config,security_model=none,mount_tag=source')
+        base_qemu_args+=("-virtfs" 'local,id=source,path='"${workdir}"'/src/config,security_model=none,mount_tag=source,multidevs=remap')
```

Comment:
Why do we need `multidevs=remap`? Maybe a comment here or in the commit message?

Reply:
Sure, I'll add that. The reason, though, is that Qemu emits a warning that its needed. This was added because of `/etc/pki` being presented along with $WORKDIR.